### PR TITLE
(PC-9704) App Native - Decli Web - Re-configuration des snapshots pou…

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
…r supporter le web et native

Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-9704

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -4,11 +4,12 @@ module.exports = {
   setupFiles: ['./jest/jest.setup.ts', './node_modules/react-native-gesture-handler/jestSetup.js'],
   setupFilesAfterEnv: ['./src/tests/setupTests.js'],
   transform: {
-    '^.+\\.(js)$': '<rootDir>/node_modules/babel-jest',
+    '^.+\\.(js)$': 'babel-jest',
   },
   transformIgnorePatterns: [
     'node_modules/(?!(jest-)?react-native' +
       '|@react-navigation' +
+      '|@react-native' +
```
Comment (BenStirrup) [inline jest.config.js:12]: c'est pas redondant avec la 1ère ligne ?

Comment (kopax-polyconseil) [reply jest.config.js:12]: non

**Review by antoinewg — COMMENTED** [review]
Ce serait plus agréable à la relécture d'avoir des commits séparés. Et est-ce qu'on a besoin de tous ces fichiers web-snap dès maintenant?

File: jest.config.js (hunk 1 above)
Comment (BenStirrup) [reply jest.config.js:12]: bon, je continue : pourquoi ?^^

**Review by antoinewg — APPROVED** [review]
LGTM

**Pull request merged**